docs/src/main/paradox/scala/http/client-side/host-level.md

```diff
@@ -1,6 +1,6 @@
 # Host-Level Client-Side API
 
-As opposed to the @ref[Connection-Level Client-Side API](connection-level.md#connection-level-api) the host-level API relieves you from manually managing individual HTTP
+As opposed to the @ref[Connection-Level Client-Side API](connection-level.md) the host-level API relieves you from manually managing individual HTTP
```

Comment:
👍, 5bcbd7552128881d4e665f045eea2987bf11c28d didn't remove all use of the `connection-level-api{,-java}` anchor.